services/AccountService.py

```diff
@@ -28,7 +22,7 @@ def read_account(self, name:string)->Account:
     #login function 
     #check if the username and password combination exist in the database
     #if not, return null object
-    def login(self, username:string, password:string)->Account:
+    def login(self, username: string, password: string) -> Account:
```

Comment:
I think this is another thing we were talking about this week, not wanting to store passwords as plain text.  We should look into some existing authentication options or the salt/hash pattern Luke mentioned.

Reply:
I agree, but I think we should come back to this on a future sprint